src/decorators/withPlaygroundRenderer.tsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
 import PlaygroundRenderer from "../components/PlaygroundRenderer/PlaygroundRenderer";
 import { StoryFn } from "@storybook/react";
+import { PlaygroundParameters } from "@/types";
+import { usePlaygroundRendererCode } from "@/hooks";
 
-const withPlaygroundRenderer: StoryFn = (_Story, { args, parameters }) => {
-  const {
-    playground: { code: parameterCode, components: parameterComponents },
-  } = parameters;
-  const code = args.code || parameterCode || { jsx: "", css: "" };
-  return <PlaygroundRenderer code={code} components={parameterComponents} />;
+const withPlaygroundRenderer: StoryFn = (_Story, { parameters }) => {
```

Comment:
```suggestion
const withPlayground: StoryFn = (_Story, { parameters }) => {
```

Reply:
Ok will consider renaming all the renderer-related to keep consistency

Reply:
@talkor mind if we'll create a different task for it? omitting the renderer?

Reply:
Of course